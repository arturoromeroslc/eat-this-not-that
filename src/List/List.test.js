import React from 'react'
import list from './list'
import renderer from 'react-test-renderer'

test('list snapshot test', () =>{
	const component = renderer.create(<list />)
	const tree = component.toJSON();
	expect(tree).toMatchSnapshot()
})
