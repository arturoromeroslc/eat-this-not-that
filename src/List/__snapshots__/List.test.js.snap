// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List snapshot test 1`] = `null`;
